data_write.py

```diff
@@ -1,122 +1,121 @@
-## This program reads in information from our assorted files and writes it into a single, easy to read and store file.
+# This program reads in information from our assorted files.
+# It then writes it into a single, easy to read and store file.
```

Comment:
```suggestion
"""
This program reads in information from our assorted files.
It then writes it into a single, easy to read and store file.
"""
```
This can be a module-level docstring.